src/arrow-sdk.ts

```diff
@@ -269,24 +272,54 @@ export async function submitOptionOrder(
     version = DEFAULT_VERSION
 ) {
     if (!isValidVersion(version)) throw UNSUPPORTED_VERSION_ERROR
+    
+    if(
+        deliverOptionParams.orderType === 3 && 
+        deliverOptionParams.payPremium === null
```

Comment:
This can change to `undefined` instead of `null` if you change `payPremium?: bool | null` to `payPremium?: bool`

Reply:
Oh, I see that you already changed `payPremium?` so you should probably update this since I don't think it will ever be null, it would only ever be `undefined` (thanks javascript)